rest-server/src/config/job.js

```diff
@@ -90,6 +90,9 @@ const jobConfigSchema = Joi.object().keys({
   killAllOnCompletedTaskNumber: Joi.number()
     .integer()
     .default(0),
+  queue: Joi.string()
```

Comment:
Do not use 'queue' here; use 'virtualCluster' instead. This is because 'queue' is a concept of Hadoop, but users only need to know 'vc', the user-level concept.